src/Uno.UI/UI/Xaml/Controls/Image/HtmlImage.wasm.cs

```diff
@@ -0,0 +1,12 @@
+﻿#nullable enable
+
+namespace Windows.UI.Xaml.Controls;
+
+public class HtmlImage : UIElement
+{
+	public HtmlImage() : base("img")
+	{
+		// Avoid the "drag effect" which is set by default in browsers
+		SetAttribute("draggable", "false");
```

Comment:
Some browsers may also require css for this like  `user-drag: none` and the prefixed versions.

https://caniuse.com/?search=user-drag

Reply:
Looks like we should set it by default in css?